Bottom: 072c803814d241b1753cce0b65732dc63fd8ffcc
Top:    6b30be6961903f3bce9e6541e0ff9e86e92019bc
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-06 01:47:13 +0100

Cure crosshair lagging on screen

FIXME: SHOULD NOT RE-WRITE THE GTK HID


---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index d3d5c16..37dd2af 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -674,9 +674,7 @@ ghid_port_window_motion_cb (GtkWidget * widget,
   queue_tooltip_update (out);
 #endif
 
-  ghid_show_crosshair (FALSE);
-  ghid_show_crosshair (TRUE);
-  if (moved && have_crosshair_attachments ())
+  if (moved)
     ghid_draw_area_update (gport, NULL);
   return FALSE;
 }
